lms/models/grouping.py

```diff
@@ -192,6 +193,11 @@ class Course(Grouping):
     )
     """Assignments that belong to this course."""
 
+    lms_course: Mapped[LMSCourse] = sa.orm.relationship(
```

Comment:
These are useful to get the reference to the new table's object based on the existing one.